packages/Modal/index.js

```diff
@@ -28,11 +28,16 @@ export function Modal({
   return (
     <>
       <Portal>
-        <DialogBackdrop onClick={closeModal} {...rest}>
+        <DialogBackdrop {...rest}>
```

Comment:
`Dialog` will automatically call `hide` when clicking outside it (even if `DialogBackdrop` is not used).

Reply:
Thank you for this @diegohaz, we really appreciate 👍
Thanks also for [reakit](https://github.com/reakit/reakit) it's such a great library that we really enjoy using!